docs/reference/run.md

```diff
@@ -629,6 +631,12 @@ command:
 
     $ docker run --security-opt label:type:svirt_apache_t -it centos bash
 
+If you want to prevent your container processes from gaining additional
+privileges, you can execute the following command:
+
+    $ docker run --security-opt no-new-privileges -it centos bash
+
+
 > **Note**: You would have to write policy defining a `svirt_apache_t` type.
```

Comment:
Can you move this above your example. Because this applies to the example above :-)